COMPILING.md

```diff
@@ -0,0 +1,161 @@
+Jamulus can be compiled for Linux, Windows and macOS as follows. Alernatively, you may wish to use one of the contributed [installation scripts](https://github.com/jamulussoftware/installscripts).
+
+---
+
+
+## Download sources
+
+For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+
+For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/master.zip)
+
+## Linux
+
+#### Install dependencies
+
+On Ubuntu-based distributions 18.04+, Debian 9+ or 10 and Raspberry Pi Raspbian Buster release or later:
+
+* Build-essential
+* Qt5-qmake
+* Qtdeclarative5-dev
+* Qt5-default
+* Qttools5-dev-tools
+* Libjack-jackd2-dev
+
+#### On Fedora
+
+* Qt5-qtdeclarative-devel
+* Jack-audio-connection-kit-dbus
+* libQt5Concurrent5
+* Jack-audio-connection-kit-devel
+
+#### For all desktop distributions
+
+[QjackCtl](https://qjackctl.sourceforge.io/) is optional, but recommended to configure JACK.
+
+
+### Standard desktop build
+
+~~~
+qmake Jamulus.pro
+make clean
+make
+sudo make install
+~~~
+
+This puts the Jamulus binary into `/usr/local/bin`.
+
+#### Notes
+
+* The "make clean" command is essential to remove the automatically generated Qt files which are present in the .tar.gz file and may not match the Qt version you are using.
+
+* To use an external shared OPUS library instead of the built-in one use qmake `"CONFIG+=opus_shared_lib" Jamulus.pro`.
+
+* To use this file configure the software with `qmake "CONFIG+=noupcasename" Jamulus.pro` to make sure the output target name of this software is **j**amulus instead of **J**amulus.
```

Comment:
What do you mean with "To use this file"?

I'm wondering if this is relevant to main stream users, I think this is rather a packaging/policy thing.

```suggestion
* The final binary will be called **J**amulus. If you prefer a lower case **j**amulus, configure the software with `qmake "CONFIG+=noupcasename" Jamulus.pro`.
```

Reply:
Maybe remove it as per the comment above?